packages/components/src/custom-select-control-v2/types.ts

```diff
@@ -14,6 +14,19 @@ export type CustomSelectStore = {
 
 export type CustomSelectContext = CustomSelectStore | undefined;
 
+type CustomSelectSize< Size = 'compact' | 'default' > = {
```

Comment:
For context, the public interface of `CustomSelectControlV2` only has these two size variants, whereas the private (internal) `CustomSelectButton` has three size variants for back compat.